src/Confluent.Kafka/Timestamp.cs

```diff
@@ -80,6 +88,15 @@ public static long DateTimeToUnixTimestampMs(DateTime dateTime)
             }
         }
 
+        /// <summary>
+        ///     Convert a unix timestamp to a DateTime value.
+        /// </summary>
+        /// <param name="timestamp">
+        ///     The unix timestamp to convert.
+        /// </param>
+        /// <returns>
+        ///     The DateTime value associated with <paramref name="timestamp"/>
+        /// </returns>
         public static DateTime UnixTimestampMsToDateTime(long timestamp)
             => new DateTime(1970, 1, 1, 0, 0, 0, 0, System.DateTimeKind.Unspecified)
```

Comment:
Oh, we've cut explicit production of timestamps in the upcoming release, so that also goes some way to explaining why there is no testing for it yet.